ethcore/builtin/Cargo.toml

```diff
@@ -14,7 +14,10 @@ keccak-hash = "0.2.0"
 log = "0.4"
 num = { version = "0.1", default-features = false, features = ["bigint"] }
 parity-bytes = "0.1"
-parity-crypto = "0.4.0"
+eip-152 = { path = "../../util/EIP-152" }
+parity-crypto = { version = "0.4.0", git = "https://github.com/paritytech/parity-common/", branch = "dp/feature/add-blake2_f-toparity-crypto"}
```

Comment:
Do we want to use an `EIP-152` crate or move it to parity-crypto? If it's not used anywhere else, having a crate in `util` is IHMO fine.

Reply:
Yes, we'll have `util/eip-152` (see https://github.com/paritytech/parity-ethereum/pull/11017/commits/263e3593e3ee2bf976b836fc2824a1d6a15728e3).

Reply:
Should this line be removed then?
```suggestion
```